### PR TITLE
tistory.com sorted to common rules

### PR DESCRIPTION
**Pull request by stanislav-atr**

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -5466,27 +5466,13 @@ dap-news.com##.ads
 @@||gmyankee.tistory.com^$generichide
 @@||hjkhgyss.tistory.com^$generichide
 @@||liveinincheon.tistory.com^$generichide
-
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=liveinincheon.tistory.com
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=hjkhgyss.tistory.com
-@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=maniac-game.tistory.com
-
-@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=viewingcat.tistory.com
+@@||daumcdn.net/*#ad&adv_keywords*impopup$domain=liveinincheon.tistory.com|hjkhgyss.tistory.com|maniac-game.tistory.com
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:5469]: Is this necessary? I don't see the blocking request. Please provide a screenshot.

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:5469]: This is just sorting a bunch of rules.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/foreign.txt:5469]: Ok.

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -5466,27 +5466,13 @@ dap-news.com##.ads
...
-plok01234.tistory.com##.adsbygoogle
-jimnong.tistory.com##.adsbygoogle
-dotobi-game.tistory.com##.adsbygoogle
-maniac-game.tistory.com##.adsbygoogle
-jwmx.tistory.com##.adsbygoogle
-viewingcat.tistory.com##.adsbygoogle
-ittopic.tistory.com##.adsbygoogle
 !#endif
+! ##.adsbygoogle
+!+ PLATFORM(ext_ff, ext_opera, ios, ext_safari, ext_android_cb, ext_ublock)
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:5474]: Can be moved to the group with directive.

Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:5474]: Many links to issues can be replaced by description of fixed issue.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Pull request merged**